### PR TITLE
added solution to day2 problems

### PR DESCRIPTION
**Pull request by nabinbogati**

### CODE REVIEW

**Comment by nabinbogati** [comment]
my solution to the day2 problems

File: Day 2 - Python Datetime objects and Timedelta/solution/solution.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from datetime import datetime, timedelta
+
+christmas_date = datetime(year=2019, month=12, day=25)
+
+# 3 weeks and 4 days time can be interpreted as 3*7days+4days = 25 days
```
Comment (pylenin) [inline Day 2 - Python Datetime objects and Timedelta/solution/solution.py:5]: There is also an argument for `weeks`. If you use that, you don't have to calculate the number of days.

File: Day 2 - Python Datetime objects and Timedelta/solution/solution.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from datetime import datetime, timedelta
```
Comment (pylenin) [inline Day 2 - Python Datetime objects and Timedelta/solution/solution.py:1]: Rename the solution file as `solution{{Qns. No.}}.py`.

**Comment by pylenin** [comment]
Please provide your full name, email and any social media profile(facebook, twitter, github etc.) in the comments on top of your solution files. That way people can know who you are.

**Comment by pylenin** [comment]
Sorry buddy. Someone already provided the solution. Try to fix your solutions more quickly next time.

**Pull request closed without merging**